mkdocs.yml

```diff
@@ -1,2 +1,39 @@
 site_name: onosproject
+site_url: https://www.onosproject.org
+site_description: Open Network Operating System (ONOS)
+site_author: Open Networking Foundation
 theme: 'material'
+
+# Repository
+repo_name: onosproject
+repo_url: https://github.com/onosproject
+
+# Page tree
+nav:
+   - ONOS: README.md
+   - Developers:
+      - Contacts and Calenders: developers/community-info.md
+      - Development Prerequisites: developers/prerequisites.md
+      - Developer Workflow: developers/dev_workflow.md
+      - Contribution Guidelines: developers/contributing.md
+   - Configuration Subsystem: 
+      - How To Build Configuration Subsystem?: onos-config/build.md 
+      - How To Run Configuration Subsystem?: onos-config/run.md
+      - How To Debug Configuration Subsystem?: onos-config/debugging.md
+      - Administrative And Diagnostic CLI: onos-config/cli.md
+      - Extending Configuration Subsystem With Model Plugins: onos-config/modelplugin.md
+      - Northbound gNMI Service: onos-config/gnmi.md
```

Comment:
Add `gNMI Extensions`

Reply:
gNMI Extensions is refereed by gnmi.md doc. Do we need to include it one more time? I considered it as a supplement document but if that is important to be separate we can add that.

Reply:
Yeah I think it should have a tree entry. Otherwise you'd have to remember how you got to it

Reply:
Done